api/resource_remote_builders.go

```diff
@@ -2,7 +2,7 @@ package api
 
 import "context"
 
-func (client *Client) EnsureMachineRemoteBuilderForApp(ctx context.Context, appName string) (*Machine, *App, error) {
+func (client *Client) EnsureRemoteBuilder(ctx context.Context, orgID string, appName string) (*Machine, *App, error) {
```

Comment:
since `orgID` and `appName` are both strings, you can compress their declaration like so: `orgID, appName string`.